buildroot/package/configtxt/configtxt.mk

```diff
@@ -10,6 +10,7 @@ define CONFIGTXT_INSTALL_TARGET_CMDS
         echo "# Enable I2C and SPI" >> $(BINARIES_DIR)/rpi-firmware/config.txt
 	echo "dtparam=i2c=on" >> $(BINARIES_DIR)/rpi-firmware/config.txt
 	echo "dtparam=spi=on" >> $(BINARIES_DIR)/rpi-firmware/config.txt
+	echo "dtoverlay=gpio-ir,gpio_pin=16" >> $(BINARIES_DIR)/rpi-firmware/config.txt
```

Comment:
Don't do this as this might break existing configurations. If a users wan't to enable it, he should add this and check first what GPIOs are available.

Reply:
You might add the line, but comment it out.

Reply:
Problem is, config.txt is not writeable when booted, so users would need to remove the sd card and change the file using a PC

Reply:
You simply need to remount the partition read/writeable
 mount -o remount,rw /boot

We can't accept a change that will break some installations.

Reply:
Well then it isn't as much of an issue. As there is still some manual configuration required by the user, adding this line to config.txt shouldn't be an issue. I'll update the docs soon.

Reply:
I've removed this line and added instructions how to add it.

What happens when a user updates? Does this line get copied over or do i need to implement some logic to add it back during updates?

Reply:
Updated usually don't overwrite config file but only edit them. I can't guarantee this will always be that way, but we try to keep it this way if possible.